NSS-Day-Cohort-68/rare-client-rare-team3
# Create a Posts Form

I created the form to create a post and added navigation to the post details page of the newly created post. This PR completes ticket #40.

To test:
1. `git fetch --all` 
2. switch to this branch `stacy/posts/create_post/client`
3. If not logged in, login as a user from your database.
4. Click on `New Post` in the nav bar
5. Fill out the post form and click save
6. The new post should be created and you should be taken to the details page of the new post
7. Lastly, the title, article content, and category are required fields. The image url is optional. With all of this in mind leave out one of the required fields to check to see if the incomplete form alert is triggered.
